prison-core/src/main/java/tech/mcprison/prison/internal/events/block/BlockBreakEvent.java

```diff
@@ -35,11 +35,13 @@
     private Location blockLocation;
     private Player player;
     private boolean canceled = false;
+    private int exp;
 
-    public BlockBreakEvent(BlockType block, Location blockLocation, Player player) {
+    public BlockBreakEvent(BlockType block, Location blockLocation, Player player,int xp) {
```

Comment:
This is a breaking change! Can you restore the old constructor along with this new one?